.github/workflows/e2e-compatibility.yaml

```diff
@@ -1,5 +1,11 @@
+# Runs compatibility tests for ibc-go.
+# Can be triggered manually by setting values for release-branch and ibc-go-version.
+# On a weekly schedule with default values of 'main' for release-branch and 'main-cron-job' for ibc-go-version.
 name: Compatibility E2E
 on:
+  schedule:
+    # run on 20:00 on Sunday.
```

Comment:
Isn't this Saturday?

Reply:
Ah I see! Super weird that they themselves link to crontab.guru which says it's Saturday :D https://crontab.guru/#0_20_*_*_6 but  thanks for checking!